src/jit/codegenlinear.cpp

```diff
@@ -1333,6 +1333,25 @@ void CodeGen::genConsumeRegs(GenTree* tree)
             // Update the life of the lcl var.
             genUpdateLife(tree);
         }
+#ifdef FEATURE_HW_INTRINSICS
+        else if (tree->OperIs(GT_HWIntrinsic))
```

Comment:
`GT_HWIntrinsic` is already handled a few lines below.

Reply:
Did this get addressed? I don't see the matching removal

Reply:
It's a bit hard to see, with the way the diffs are shown, but if you look below, this line was deleted:
```
        else if (tree->OperIsHWIntrinsic())
```
and the `genConsumeReg` below it is actually the one for the `OperIsInitVal` case.